client/datasync-client/package.json

```diff
@@ -22,6 +22,11 @@
     "require": [
       "ts-node/register"
     ],
+    "reporter": [
+      "lcov",
+      "text"
```

Comment:
Lack of consistency - in some cases we using only `lcov` as reporter. Less is better :D